weatherrouting/routers/linearbestisorouter.py

```diff
@@ -32,7 +32,11 @@ def route (self, lastlog, time, start, end):
 		position = start
 		path = []
 		for p in isoc[-1]:
-			if utils.pointDistance (end[0],end[1], p[0], p[1]) < self.getParamValue('minIncrease'):
+			(twd,tws) = self.grib.getWindAt (time + datetime.timedelta(hours=1), p[0], p[1])
+			maxReachDistance = self.polar.maxReachDistance(p, twd, tws)
+			if utils.pointDistance (end[0],end[1], p[0], p[1]) < maxReachDistance*1.1:
```

Comment:
This should use the minIncrease value, since routing should be customizable for that

Reply:
see issue reported below